bigtop.bom

```diff
@@ -86,7 +86,10 @@
        packaging = 'rpm' // *optional* If this component can be built only as either
                          // DEB or RPM, specify that packaging format explicitly.
                          // If both formats are supported, omit this option.
-       maven_parallel_build = true
+       maven_parallel_build = true // *optional* maven_parallel_build indicates whether the component can be built in parallel.
+                                   // Components with this flag can have the Maven parallel build parameter -PbuildThreads=2C or -PbuildThreads=2
```

Comment:
Currently `-PbuildThread=2` is not allowed. How about fix [the code](https://github.com/apache/bigtop/blob/fc93b76a8d3f51531af730f9f6397e91fd6b91cb/packages.gradle#L456-L457) here? @JiaLiangC

Reply:
Oops. My comment conflicts with @guyuqi. We can fix this with follow-up JIRA.

Reply:
> Currently `-PbuildThread=2` is not allowed. How about fix [the code](https://github.com/apache/bigtop/blob/fc93b76a8d3f51531af730f9f6397e91fd6b91cb/packages.gradle#L456-L457) here? @JiaLiangC

@iwasakims  Thank you for the reminder. I will fix the code.